### DISCUSSION TITLE
Windows 10 taskbar search box long and short

### DISCUSSION THREAD

**Discussion by XuanChen1234** [valinet/ExplorerPatcher#1739] [Ideas]
I was thinking of adding Windows 10 taskbar show search box long or short options.
![屏幕截图 2023-07-28 110344](https://github.com/valinet/ExplorerPatcher/assets/140480110/3026fb45-803f-4418-bb92-c025db3a1480)

**Comment by milestprower92** [comment]
No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the old search box from the old taskbar and there are no plans to have it added back due to the fact that the search icon does the same thing as the search box.
